### PR TITLE
[Bug 1453140] Add Download Firefox button on Firefox pages like mozilla.org

### PR DESCRIPTION
**Pull request by mlagace**
* Added a Download Firefox button on Firefox product/documents/wiki pages that matches the same style as on mozilla.org.

image

### CODE REVIEW

**Comment by jpetto** [comment]
I'm guessing this will be moot after #3243 merges, but just calling out the difference in button colors here:

screen shot 2018-06-18 at 3 06 12 pm

File: kitsune/sumo/static/sumo/less/products.less [hunk 1]
```diff
@@ -3,95 +3,13 @@
...
-    display: block;
-    line-height: 20px;
-    position: relative;
-    text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.25);
-    transition: top 0s;
-    transition: all 0.5s;
-    width: 230px;
-
-     .download-content {
-       background: url('../img/firefox-small.png') no-repeat;
```
[the remark this answers is not in the document]
Comment (mlagace) [reply kitsune/sumo/static/sumo/less/products.less]: I didn't find that image used anywhere else so I removed it.

**Comment by mlagace** [comment]
Rebased

**Comment by jpetto** [comment]
The plight of a batch of semi-related PRs. One more rebase?

**Pull request merged**